### PR TITLE
Cleaning up generator-ui-component to meet the new HX standards

### PR DESCRIPTION
**Pull request by manifestinteractive**
#### What does this PR do?

The previous version of the `generator-ui-component`  no longer creates components the way HX uses them.  This PR updates the generator to generate the components the way we are now using them.

#### What tests does this PR have?

Run `npm test` from the root to test this tool.

#### How can be this tested?

To test the tool, read the README.md file on how to create a new component using Yeoman.

#### Screenshots / Screencast

N/A

#### What gif best describes how you feel about this work?

Celebrate

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

### CODE REVIEW

File: generators/base/index.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ module.exports = generators.Base.extend({
 
   _generateFileName: function(input, ext) {
     if(!input) return '';
-    return [_.snakeCase(input), (ext || 'jsx')].join('.')
+    return [_.camelCase(input), (ext || 'jsx')].join('.')
```
Comment (lukehansell-hx) [inline generators/base/index.js]: Tripapp standard is snake_case, UI-Toolkit standard is camelCase. Which should we settle on?

**Comment by lukehansell-hx** [comment]
SE/Guru

**Comment by lukehansell-hx** [comment]
:ring:

**Comment by jodiedoubleday** [comment]
I think every other project is camelCase except tripApp, so lets stick with camelCase - Any objections?
@jackdcrawford @Joezo

**Comment by jackdcrawford** [comment]
:camel: :+1:

**Comment by jodiedoubleday** [comment]
:+1:

**Pull request merged**